modules/ref-inference-endpoints.adoc

```diff
@@ -216,6 +216,15 @@ endif::[]
 * `v2`
 --
 
+[NOTE]
+====
+ModelMesh does not support the following REST endpoints:
+
+* `v2/health/live`
+* `v2/health/ready`
+* `v2/models/<model_name>[/versions/]/ready`
```

Comment:
* `v2/models/<model_name>[/versions/]/ready`

Does the plural versions imply that multiple version numbers can appear in the endpoint consecutively, or does it imply that only one version of multiple versions can appear?

Reply:
It might not actually be a placeholder. The square brackets confused me a bit.

Reply:
Per separate discussion - we'll add this as-is for the moment, and follow up in the new year on whether the formatting of `/versions/` needs any improvement.